backstory/src/main/java/com/google/sps/servlets/PromptManager.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//s
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.sps.servlets;
+
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+public final class PromptManager {
+
+  private List<String> keywords;
+  
+  /**
+   * Initialize labels parameter
+   * 
+   * @param labels A list of Strings containing keywords for prompts.
+   */
+  public PromptManager(List<String> keywords){
+    this.keywords = keywords;
+  }
+
+  /**
+   * Generates prompt using labels.
+   * 
+   * @return A String containing the output prompt.
+   */
+  public String generatePrompt() {
+    String prompt = "";
+    if(keywords.size() == 0){
+        return "";
+    }
+
+
+    for(int incrementer = 0; incrementer < keywords.size() - 1; incrementer++){
+        prompt = prompt + keywords.get(incrementer) + " ";
```

Comment:
Consider changing the delimiter character (" ") to be a parameter of the function, then you could get easily labels like:
"label1 and label2 and label3 ... " if we decide that would be slightly better for the prompt.

Reply:
Done